@xviz/server can't install

```
hooke@DESKTOP-8207B70:/mnt/g/code/hetu.gl$ npm i @xviz/server
npm WARN @megvii/hetu.gl@1.2.0 requires a peer of typescript@>=3.7 but none is installed. You must install peer dependencies yourself.
npm WARN @luma.gl/addons@7.3.2 requires a peer of @luma.gl/core@^7.1.0 but none is installed. You must install peer dependencies yourself.
npm WARN orbit-controls-es6@2.0.1 requires a peer of three@0.108.0 but none is installed. You must install peer dependencies yourself.
npm WARN three.interaction@0.2.2 requires a peer of three@^0.89.0 but none is installed. You must install peer dependencies yourself.
npm WARN @megvii/hetu.gl@1.2.0 No license field.
npm WARN The package @types/node is included as both a dev and production dependency.
npm WARN The package copy-webpack-plugin is included as both a dev and production dependency.
npm WARN The package semantic-release is included as both a dev and production dependency.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /mnt/g/code/hetu.gl/node_modules/@xviz/server/bin/xvizserver
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/mnt/g/code/hetu.gl/node_modules/@xviz/server/bin/xvizserver'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/hooke/.npm/_logs/2019-12-05T12_19_26_016Z-debug.log

hooke@DESKTOP-8207B70:/mnt/g/code/hetu.gl$ node -v
v12.9.1

hooke@DESKTOP-8207B70:/mnt/g/code/hetu.gl$ npm -v
6.13.1

```

--- a/docs/developers-guide/validating-xviz.md
+++ b/docs/developers-guide/validating-xviz.md
@@ -23,6 +23,7 @@ import {XVIZBuilder} from '@xviz/builder';
 
 const builder = new XVIZBuilder();
 
+// prettier-ignore
 builder
   .pose()
   .timestamp(ts1);
@@ -36,5 +37,5 @@ import {XVIZValidator} from '@xviz/schema';
 
 const validator = new XVIZValidator();
 
-validator.validateStateUpdate(builder.getMessage())
+validator.validateStateUpdate(builder.getMessage());
 ```

--- a/modules/server/package.json
+++ b/modules/server/package.json
@@ -8,13 +8,14 @@
     "url": "https://github.com/uber/xviz.git"
   },
   "bin": {
-    "xviz": "./bin/xvizserver"
+    "xvizserver": "./bin/xvizserver"
   },
   "main": "dist/es5/index.js",
   "module": "dist/esm/index.js",
   "esnext": "dist/es6/index.js",
   "files": [
-    "dist"
+    "dist",
+    "bin/xvizserver"
   ],
   "browser": {
     "text-encoding": false

